### ISSUE TITLE
hasCollections is always false

### ISSUE THREAD

**Issue by cuddlyogre** [TobyLobster/ImportLDraw#36]
I'm on revision 82e6764 and blender-2.80.0-git.ebc44aae9897-windows64.

hasCollections is set on line 236 but is always false. 
`hasCollections = hasattr(bpy.data, "collections")`

This causes the error with `setupInstructionsLook`
```
    if bpy.data.groups.find('Black Edged Bricks Collection') < 0:
AttributeError: 'BlendData' object has no attribute 'groups'
```

When checking in the Python console, that function returns true and when calling the function where `hasCollections` is used, it also returns true.

To troubleshoot this, I used this piece of code where `hasCollections` is defined.
```
for obj in bpy.data.objects:
    print(obj.name)
```

This is the error returned when Blender is started:
```
    for obj in bpy.data.objects:
AttributeError: '_RestrictData' object has no attribute 'objects'
```

The cause is described at https://blenderartists.org/t/attributeerror-restrictdata-object-has-no-attribute-filepath/607717 but I was unable to come up with a solution that would work with ImportLDraw.

**Comment by cuddlyogre** [comment]
This is fixed in 265008c52e51860e0e2a20026463e80899d8be46

**Comment by cuddlyogre** [comment]
This is corrected in ff6dff28833d364c60553e16f25fa4839b9d2b8b

**Comment by TobyLobster** [comment]
Merged with thanks.
